http-add-on/templates/operator/deployment.yaml

```diff
@@ -29,12 +29,7 @@ spec:
         - --v=10
         image: "{{ .Values.images.kubeRbacProxy.name }}:{{ .Values.images.kubeRbacProxy.tag }}"
         resources:
-            limits:
-              cpu: 300m
-              memory: 200Mi
-            requests:
-              cpu: 10m
-              memory: 20Mi
```

Comment:
These configs are for the RbacProxy, not for the operator itself. The operator is already reading the config from values. I agree with adding parameters for configuring these options, but they have to be split from operator as this is just for metrics

Reply:
Thanks for this review here as well! The requested updates have been made. This is failing the CI process though the same as the other PR I have open:

https://github.com/kedacore/charts/pull/568#discussion_r1408158455

Reply:
yeah, let's wait until we merge [the fix for the CI](https://github.com/kedacore/charts/pull/582)